soc/arm/nxp_lpc/lpc55xxx/Kconfig.soc

```diff
@@ -151,4 +151,12 @@ config LPC55XXX_SRAM_CLOCKS
 	  By default, CMSIS SystemInit will enable the clock to these RAM banks.
 	  Disable this Kconfig to leave the ram banks untouched out of reset.
 
+config LPC55XXX_USB_RAM
+	bool "Is USB RAM Present?"
```

Comment:
I think this kconfig should not have a menu prompt. Also, would this be more appropriate as a devicetree property?

Reply:
Removed the menu prompt. This is Kconfig as we need this to be available in the CMake file.